### PR TITLE
build: upgrade client dependencies

### PR DESCRIPTION
**Pull request by grianbrcom**

### CODE REVIEW

**Review by temanbrcom — APPROVED** [review]
🌮

File: clients/cobol-lsp-vscode-extension/package.json [hunk 1]
```diff
@@ -199,24 +199,26 @@
         "vscode:prepublish": "npm run compile",
         "compile": "tsc -p ./",
         "watch": "tsc -watch -p ./",
-        "postinstall": "node ./node_modules/vscode/bin/install",
         "test": "jest -w 1 --unhandled-rejections=strict",
         "coverage": "jest --coverage --runInBand"
     },
     "dependencies": {
-        "@zowe/imperative": "4.7.4",
+        "@zowe/imperative": "4.13.1",
```
Comment (asatklichov) [inline clients/cobol-lsp-vscode-extension/package.json:206]: What is output with `>npm outdated     `   command, please just upload snapshot here, so will be helpful for later

**Review by asatklichov — APPROVED** [review]
🦝

File: clients/cobol-lsp-vscode-extension/package.json (hunk 1 above)
Comment (grianbrcom) [reply clients/cobol-lsp-vscode-extension/package.json:206]: Good idea!
For Server:
```
[INFO]   ch.qos.logback:logback-classic ................. 1.2.3 -> 1.3.0-alpha5
[INFO]   com.google.guava:guava ........................ 29.0-jre -> 30.1.1-jre
[INFO]   com.google.inject:guice ............................... 4.2.2 -> 5.0.1
[INFO]   com.google.inject.extensions:guice-assistedinject ..... 4.2.2 -> 5.0.1
[INFO]   commons-io:commons-io ................................... 2.5 -> 2.9.0
[INFO]   org.antlr:antlr4 ...................................... 4.7.1 -> 4.9.2
[INFO]   org.antlr:antlr4-maven-plugin ......................... 4.7.1 -> 4.9.2
[INFO]   org.antlr:antlr4-runtime .............................. 4.7.1 -> 4.9.2
[INFO]   org.apache.commons:commons-lang3 ....................... 3.9 -> 3.12.0
[INFO]   org.apache.commons:commons-text ........................... 1.6 -> 1.9
[INFO]   org.awaitility:awaitility ............................. 3.0.0 -> 4.1.0
[INFO]   org.eclipse.lsp4j:org.eclipse.lsp4j .................. 0.5.0 -> 0.12.0
[INFO]   org.eclipse.lsp4j:org.eclipse.lsp4j.jsonrpc .......... 0.5.0 -> 0.12.0
[INFO]   org.junit.jupiter:junit-jupiter-engine ............. 5.6.0 -> 5.8.0-M1
[INFO]   org.junit.jupiter:junit-jupiter-params ............. 5.6.0 -> 5.8.0-M1
[INFO]   org.junit.platform:junit-platform-runner ........... 1.6.0 -> 1.8.0-M1
[INFO]   org.mockito:mockito-core ............................ 2.23.0 -> 3.10.0
[INFO]   org.mockito:mockito-junit-jupiter ................... 2.23.0 -> 3.10.0
[INFO]   org.projectlombok:lombok .......................... 1.18.10 -> 1.18.20
[INFO]   org.slf4j:slf4j-api ........................... 1.7.25 -> 2.0.0-alpha1
```
For Client:
```
Package                     Current   Wanted   Latest  Location  Depended by
@zowe/imperative            MISSING   4.13.1   4.13.1  -         cobol-lsp-vscode-extension
json-schema                 MISSING    0.3.0    0.3.0  -         cobol-lsp-vscode-extension
lodash                      MISSING  4.17.21  4.17.21  -         cobol-lsp-vscode-extension
vscode-extension-telemetry  MISSING    0.1.6    0.1.7  -         cobol-lsp-vscode-extension
vscode-languageclient       MISSING    5.2.1    7.0.0  -         cobol-lsp-vscode-extension
```

**Pull request merged**